Tutorial.md

```diff
@@ -0,0 +1,45 @@
++Before you start the tutorial, we will assume that you have git installed onto your computer.
+-test	+If you don’t have git, then you can learn how to install it [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). Here is a guide on the commands of [git also.](http://feedhenry.org/student-help-guide/topic04-git-introduction/book-git-tutorial/index.html#/Basics)
+-K.H	
+ 	 
+-This is a screenshot for a tutorial	+Before you can add your own changes to the school website, you first need to “fork” the repo of which that you want to change. To “fork” a repo means to get your own copy of the repo, in this case our school website.  
+-:)	
+ 	 
++![cap1](https://i.imgur.com/rzHlCWQ.png)
```

Comment:
Will these images remain in this location long term?

Reply:
the images will remain there as long as imgur stays up, should I push them to the repo?